internal/cmd/generate/commands/genstruct/command.go

```diff
@@ -239,6 +239,11 @@ type API struct {
 			name := strings.ReplaceAll(e.Name(), "Request", "")
 			if strings.HasPrefix(strings.ToLower(name), strings.ToLower(n)) {
 				methodName := strings.ReplaceAll(name, n, "")
+				// Some methods are equal to the namespace (like 'rollup.rollup')
+				// and we don't want to have an empty string here.
+				if len(methodName) == 0 {
+				    methodName = strings.Replace(name, n, "", 1)
```

Comment:
What if we just always used `methodName = strings.Replace(name, n, "", 1)`, i.e. always replace the first occurrence, since this is what we do anyway if the name appears several times?

Reply:
It's because there's always a prefix and sometimes the namespace appears within the methodName as well (ie `RollupGetRollupJob` -> `GetJob`